examples/base_1cc/variables.tf

```diff
@@ -249,3 +249,9 @@ variable "support_access_enabled" {
   description = "If Network Security Group is being configured, enable a specific outbound rule for Cloud Connector to be able to establish connectivity for Zscaler support access. Default is true"
   default     = true
 }
+
+variable "zssupport_server" {
+  type        = string
+  description = "destination IP address of Zscaler Support access server. IP resolution of remotesupport.<zscaler_customer_cloud>.net"
+  default     = "199.168.148.101" #for commercial clouds
```

Comment:
Is it expected that this IP is different for gov clouds?

Reply:
Correct. remotesupport.<zscaler_customer_cloud>.net <<< 199.168.148.101 is the resolved IP for all commercial clouds.

This is specified in the variable description. ZSEC handles this automatically doing an fqdn lookup by cloud so if gov will update the nsg rule IP

else
                    echo "Resolving remotesupport.$zscaler_cloud to IP for Network Security Group rule..."
                    support_server_ip=$(dig +short remotesupport.$zscaler_cloud)
                    echo "${GREEN}Outbound rule permitting TCP/12002 access to $support_server_ip will be created${RESET}"
                    echo "export TF_VAR_zssupport_server='$support_server_ip'" >> .zsecrc
                fi